cli/logout.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/alibaba/pouch/credential"
+	"github.com/spf13/cobra"
+)
+
+// logoutDescription is used to describe logout command and auto generate command doc.
+var logoutDescription = "\nlogout from a v1/v2 registry."
+
+// LogoutCommand use to implement 'logout' command.
+type LogoutCommand struct {
+	baseCommand
+}
+
+// Init initialize logout command.
+func (l *LogoutCommand) Init(c *Cli) {
+	l.cli = c
+
+	l.cmd = &cobra.Command{
+		Use:   "logout [SERVER]",
+		Short: "Logout from a registry",
+		Long:  logoutDescription,
+		Args:  cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return l.runLogout(args)
+		},
+		Example: logoutExample(),
+	}
+}
+
+// runLogout is the entry of logout command.
+func (l *LogoutCommand) runLogout(args []string) error {
+	var serverAddress string
+	if len(args) > 0 {
+		serverAddress = args[0]
+	}
+
+	// TODO: better way to get registry server address, or get it from a default variable
+	registry := serverAddress
+	if registry == "" {
+		registry = "https://index.docker.io"
+	}
+
+	if !credential.Exist(serverAddress) {
+		fmt.Fprintf(os.Stdout, "Has not logged in registry: %s\n", registry)
+		return nil
```

Comment:
If this, cli will return a status code 0 which means no error. Is it reasonable? @Ace-Tang

Reply:
if registry never login before, then the logout operation should do nothing, and return nil. WDYT, @allencloud

Reply:
To be honest, I am wondering if I got all the context, but I do not think return nil is a good way, because we should let command operator be notified that this serverAddress is never logined. If one has login to registry registry.alibaba.com, and he wishes to logout, however he accidentally typed in an incorrect registry name like `registryyyy.alibaba.com`, I think we should return error rather than simply return nil.

Reply:
any string like `registryyyy.alibaba.com` will be recognized as a registry name, since the name is valid, and it has no common way to valid if a string is valid. We just know that `registryyyy.alibaba.com` has not logged in, and do nothing.

Reply:
> We just know that registryyyy.alibaba.com has not logged in

Should this be a warning message or error message? User wish to logout `registry.alibaba.com`, but type in `registryyyy.alibaba.com` by mistake, If it returns exit code 0, user would think that `registry.alibaba.com` has been logout. Does this reasonable?

Reply:
it will get a message like `Has not logged in registry: %s\n`, I can make it a warning. But whether user `type registryyyy.alibaba.com by mistake` we have no idea.

Reply:
Yes, I totally agree with the message, it tells user what exactly happened. Then could we make it exit code non-zero?

Reply:
it comes back to the beginning , I think the exit be 0 is make sense, cause it got no error

Reply:
how about one would get the status via exit code. If it is a warning message, I think we still could consider using non-zero to represent the result.

Reply:
yes, let me take back the warning word, I still think it is a info message.  we do nothing if there is no login before. The exit code should be 0 to make sense.

Reply:
OK, let me keep the remaining. 😄